src/main/groovy/org/notlocalhost/gradle/CalabashTestPlugin.groovy

```diff
@@ -54,9 +54,6 @@ class CalabashTestPlugin implements Plugin<Project> {
             project.logger.debug "${project.getPath()}"
             project.logger.debug "==========================="
 
-            def outFile = new File(project.file("build/reports/calabash/${variationName}"), "report.html")
```

Comment:
It still be referenced by `println "\r\nCalabash HTML Report: file://$outFile.canonicalPath"`